### PR TITLE
DSK-31271: Remove all PerkinElmer references

### PR DESCRIPTION
**Pull request by stein-wang**

### CODE REVIEW

File: Examples/ChemDrawOAuthExample/src/components/App.tsx [hunk 1]
```diff
@@ -75,8 +83,7 @@ class App extends React.Component<any, IAppState> {
     );
 
     // Construct a URL for authorization containing the callback key
-    const dropboxOAuthUrl =
-      `https://www.dropbox.com/oauth2/authorize?client_id=${this.appKey}&response_type=token&redirect_uri=com.perkinelmer.chemdraw.addin:///&state=${callbackKey}`;
+    const dropboxOAuthUrl = `https://www.dropbox.com/oauth2/authorize?client_id=${this.appKey}&response_type=token&redirect_uri=com.revvity.chemdraw.addin:///&state=${callbackKey}`;
```
Comment (stein-wang) [inline Examples/ChemDrawOAuthExample/src/components/App.tsx:86]: Changed the URL scheme

File: Examples/Images/open-url.png
Comment (stein-wang) [inline Examples/Images/open-url.png:1]: Updated the screenshot as the placeholder gets changed.

File: Examples/Open URL/main.html [hunk 2]
```diff
@@ -1,59 +1,70 @@
...
+          <br />
+        </div>
+      </div>
+      <div class="row form-group">
+        <div class="col-xs-12">
+          <input
+            id="url"
+            class="form-control"
+            value="https://revvitysignals.com/products/research/chemdraw"
+            type="text"
```
Comment (stein-wang) [inline Examples/Open URL/main.html:52]: Changed the placeholder text

File: README.md [hunk 3]
```diff
@@ -1,15 +1,10 @@
...
 ## Prerequisites
 
-### Windows
-
-You must install [ChemOffice Professional](http://www.perkinelmer.com/product/chemoffice-professional-chemofficepro) 17.1 or above to install and run the add-in examples.
-
-### Mac OS X
-
-You must install [ChemDraw Professional](http://www.perkinelmer.com/product/chemdraw-professional-chemdrawpro) 17.1 or above to install and run the add-in examples.
+You must install [ChemOffice](https://revvitysignals.com/products/research/chemdraw) 17.1 or above to install and run the add-in examples.
```
Comment (stein-wang) [inline README.md:7]: According to the datasheet only ChemOffice and above supports add-ins, and ChemOffice has both win and mac versions. Thus removed the instruction for different platforms. https://image.info.revvitysignals.com/lib/fe3211747364047c761777/m/1/5e8b16cf-91bd-456e-9394-a79d3df77eff.pdf?elqTrackId=9c53448a38964278b21d3f4578683e2d&elqaid=10141&elqat=2&_gl=1*1kssuzy*_ga*MzcwMTIwMDMuMTcwNTMyMzc5NQ..*_ga_F7XGE57X1J*MTcwNTMyMzc5NS4xLjEuMTcwNTMyMzk5My41OS4wLjEwNzA4MTExMTU.

File: Examples/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # ChemDraw Add-in Examples
 
-Welcome to the ChemDraw add-in examples! This directory contains all the examples used in the [ChemDraw Add-in Developer Guide](https://github.com/PerkinElmer/ChemDraw-AddIns/tree/master/Documentation).
+Welcome to the ChemDraw add-in examples! This directory contains all the examples used in the ChemDraw Add-in Developer Guide.
```
Comment (stein-wang) [inline Examples/README.md:3]: We may need to add the correct repo URL back once we have our Revvity repos ready.

**Comment by stein-wang** [comment]
Apart from the changes stated explicitly, all the changes are tidied by Prettier automatically.

File: README.md (hunk 3 above)
Comment (stevebaxter) [reply README.md:7]: I don't think ChemOffice exists now. You might just want to say to install ChemDraw and mention that not all ChemDraw product levels support add-ins?

**Review by stevebaxter — DISMISSED** [review]

File: README.md (hunk 3 above)
Comment (stein-wang) [reply README.md]: Updated. Please review again.

**Review by stevebaxter — APPROVED** [review]

**Pull request merged**